db/views/group_lesson_summaries_v04.sql

```diff
@@ -0,0 +1,15 @@
+SELECT	  l.id AS lesson_id,
+            l.date AS lesson_date,
+            gr.id AS group_id,
+            gr.chapter_id AS chapter_id,
+            slu.subject_id,
+            CONCAT(gr.group_name, ' - ', c.chapter_name) AS group_chapter_name,
+            ROUND(CAST(AVG(average_mark) AS numeric), 2)::FLOAT AS average_mark,
+            CAST(SUM(grade_count) AS bigint) AS grade_count
+FROM lessons AS l
```

Comment:
This query is pretty slow and I think including `lessons` in it makes it a lot slower. I don't think there's any need to have lessons, all the lesson information is already included in `student_lesson_summaries`